template/src/project_name/accounts/models.py

```diff
@@ -0,0 +1,7 @@
+from django.contrib.auth.models import AbstractUser
+
+class CustomUser(AbstractUser):
```

Comment:
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** We could just call this `User`. 

I'm not sure calling it `CustomUser` is helpful. It implies there is a non-custom `User` which is true in the context of Django as a whole, but not really in the context of just the project in which this model exists. ‏